engine/poly.sc

```diff
@@ -229,6 +229,29 @@ PTMidiOp : PTOp {
 	}
 }
 
+PTCrowOut : PTOp {
+
+	*new {
+	    ^super.newCopyArgs("CROW", 2);
+	}
+
+	rate { |args, resources|
+		^\control;
+	}
+
+	instantiate { |args, resources|
+	    var iargs = PTOp.instantiateAll(args);
+	    // n is crow output #
+	    var n = iargs[0];
+	    // v is voltage v
+	    var v = iargs[1];
+	    // s is slew ... hardcoded for now
+	    var s = 0.01;
+	    ^SendReply.kr(Impulse.kr(0), "/crow/out", values: [n, s, v]);
```

Comment:
Here is where you want to replace `Impulse.kr(0)` with another argument. I suggest the first arg (`args[0]`) be the trigger, and shift everything else up one.